src/NLPL.xml

```diff
@@ -0,0 +1,35 @@
+<spdx name="No Limit Public License" identifier="NLPL" osi-approved="false">
+  <urls>
+    <url>https://fedoraproject.org/wiki/Licensing/NLPL</url>
+  </urls>
+  <license>
+    <title>
+      <p>NO LIMIT PUBLIC LICENSE 
+        <br/>Version 0, June 2012 
+      </p>
+    </title>
+    <copyright>
```

Comment:
I could go either way. It seems like it's intended either to be an author statement or a copyright statement.

We could take an empirical approach: see what kind of use this license has, and what percentages include/exclude this section.